src/pkg/generator/IGenerator.go

```diff
@@ -41,6 +43,7 @@ func Process(config []byte, generators []Generator) error {
 	initialClusterSetupVals["env"] = conf["env"]
 	clusterSetupCheckpointVals["env"] = conf["env"]
 	clusterApplicationsVals["env"] = conf["env"]
+	alertManagerConfigVals["env"] = conf["env"]
```

Comment:
warum setzt du hier extra noch einmal den Wert "env" aus der Config? Wird soweit ich es gesehen habe, in der default-alertmanager-config.yaml.tpl nicht verwendet.